internal/cmd/install.go

```diff
@@ -2,18 +2,15 @@ package cmd
 
 import (
 	"context"
-	"os"
-
-	"go.uber.org/zap"
-
 	"github.com/aveloper/blog/internal/config"
 	"github.com/aveloper/blog/internal/db"
 	"github.com/aveloper/blog/internal/logger"
 	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+	"os"
 )
 
 func install(cmd *cobra.Command, _ []string) {
-	// TODO: Figure out how force will work and implement
```

Comment:
Don't remove this TODO. There are others things to be done here like resetting systemd settings etc once they are implemented.

Reply:
okay,